homeassistant/components/mqtt/mixins.py

```diff
@@ -295,32 +294,17 @@ async def _async_discover_entities() -> None:
 async def async_get_platform_config_from_yaml(
     hass: HomeAssistant,
     platform_domain: str,
-    schema: vol.Schema,
     config_yaml: ConfigType = None,
 ) -> list[ConfigType]:
     """Return a list of validated configurations for the domain."""
 
-    def async_validate_config(
```

Comment:
Removed config validation completely, since the config is already validated with the improved `CONFIG_SCHEMA`